docker-images/jmxtrans/jmxtrans_liveness_check.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+
+if [ -z "$1" ]; then
+    echo "No kafka bootstrap has been given"
+    exit 1
+else
+    export KAFKA_HEADLESS_SERVICE="$1"
+fi
+
+if [ -z "$2" ]; then
+  echo "No kafka bootstrap port given"
+else
+    export KAFKA_METRICS_PORT="$2"
+fi
+
+nc -z "$KAFKA_HEADLESS_SERVICE" "$KAFKA_METRICS_PORT"
+if [ "$?" -ne 0 ]; then
+    echo "Couldn't connect to $KAFKA_HEADLESS_SERVICE:$KAFKA_METRICS_PORT"
```

Comment:
I do not think liveness should be bound to state of some other pod. I can imagine it for readiness, but liveness should definitely describe the state of this pod only.